### ISSUE TITLE
Re-Add Custom views (calendar / tree) for orders

### ISSUE THREAD

**Issue by SchrodingersGat** [inventree/InvenTree#5320]
Re-implement "calendar" view for the various order types, in PUI

**Comment by matmair** [comment]
I think fullcalender would be optimal for the calender part. Here is a sample with typescript: https://github.com/fullcalendar/fullcalendar-examples/blob/main/react-typescript/src/DemoApp.tsx

**Comment by SchrodingersGat** [comment]
Nice, that keeps it close to what we are already doing :)

**Comment by matmair** [comment]
@SchrodingersGat is this still valid / open?

**Comment by SchrodingersGat** [comment]
Yes still planning to implement this (or happy for someone else to)
